src/actions/appActions.js

```diff
@@ -11,29 +13,26 @@ export function selectCluster(clusterID) {
  * @param {String} clusterId Cluster ID
  */
 export function loadApps(clusterId) {
+  console.log('loading1');
   return function(dispatch, getState) {
-    // This method is going to work for NP clusters, now in local dev it is not
-    // working, so early return if the cluster is a NP one.
+    const appsApi = new GiantSwarm.AppsApi();
+
     const nodePoolsClusters = getState().entities.clusters.nodePoolsClusters;
     const isNodePoolsCluster = nodePoolsClusters.includes(clusterId);
+
+    let getClusterAppsFunc;
```

Comment:
Here we decide which api method to call, the v5 or the v4 version, based on whether or not this is a nodepool cluster.

A similar pattern is applied to all the other actions.